api/src/utils/get-snapshot-diff.ts

```diff
@@ -87,19 +85,66 @@ export function getSnapshotDiff(current: Snapshot, after: Snapshot): SnapshotDif
 					diff: deepDiff.diff(undefined, sanitizeField(afterField)),
 				})),
 		].filter((obj) => Array.isArray(obj.diff)) as SnapshotDiff['fields'],
+		systemFields: [
+			...(current.systemFields ?? []).map((currentSystemField) => {
+				const afterSystemField = after.systemFields.find(
```

Comment:
```suggestion
				const afterSystemField = (after.systemFields ?? []).find(
```

This will error if no systemFields are provided and `force` is true